doc/models/aeif.rst

```diff
@@ -0,0 +1,5 @@
+Phenomological (AEIF) neuron models
```

Comment:
```suggestion
Adaptive Exponential Integrate and Fire (aEIF) neuron models
```

Reply:
```suggestion
Adpaptive exponential integrate and fire (AEIF) neuron models
```

Just adjusting the capitalization

Reply:
```suggestion
Adaptive Exponential Integrate-and-Fire (aEIF) neuron models
```
Unfortunately, though it may look surprising, "aEIF" is the proper way to write it (see e.g. [here](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3325187/?tool=pmcentrez&report=abstract)).
Similarly, "Integrate-and-Fire" should be uppercase (and I forgot the hyphens) and is abbreviated IF.

Reply:
Ok, well it looks to me that integrate-and-fire is written in the literature both ways. If the preference is uppercase, that's fine, but this needs to be consistent in nest (I think another PR in the future is in order for proper naming of things).  As for IF instead IAF, this is a bit confusing because I see IAF in nest a lot, and I dont think I've ever seen IF used.

Reply:
I have no real preference for the uppercase in general, I just though it was the normal way to go when we are defining abbreviations. Agreed for the proper naming PR

Reply:
@jessica-mitchell so are you OK with my 2nd proposal for the title?

Reply:
Yes, sorry I think I skipped over this by accident